src/lib/wagmi-config.ts

```diff
@@ -24,4 +26,9 @@ export const wagmiConfig = createConfig({
   client({ chain }) {
     return viemClientByNetworkId[chain.id]!
   },
+  connectors: [
+    walletConnect({
+      projectId: getConfig().REACT_APP_WALLET_CONNECT_TO_DAPPS_PROJECT_ID,
+    }),
+  ],
```

Comment:
That's if we want to support wc through Rainbowkit, though doesn't seem to work, I believe we need to use rainbowkit's `[getDefaultConfig](https://www.rainbowkit.com/docs/installation#configure)` instead